Sources/TUSKit/TUSAPI.swift

```diff
@@ -92,25 +92,25 @@ final class TUSAPI {
     }
     
     func makeCreateRequest(metaData: UploadMetadata) -> URLRequest {
-        func makeMetadataHeaders() -> [String: String] {
-            let fileName = metaData.filePath.lastPathComponent
+        func makeUploadMetaHeader() -> [String: String] {
```

Comment:
Naming was confusing here, as this method only creates the Upload-MetaData header with a dictionary as a value.